### ISSUE TITLE
Naked NPC's

### ISSUE THREAD

**Issue by lleyton05** [tiltedphoques/bug-feature-reports#11]
Occasionally, non quest npc's will be naked, not sure what causes this.

**Comment by Boogen222** [comment]
This was happening for quest NPCs and non quest NPCs for me and my friend as well

**Comment by tmanochehri** [comment]
Yep, same with me. Seemingly random NPCs that are naked. Sometimes my friend and I see the same naked NPC, but most of the time our games are impacted differently.
